src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1975,54 +1975,79 @@ limitations under the License.
+            <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Channel" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Audio Output" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Low Power" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Wake on LAN" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Target Navigator" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+        </clusters>
+    </deviceType>
+    <deviceType>
+        <name>MA-casting-videoclient</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Casting Video Client</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0029</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Media Playback" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Content Launcher" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Keypad Input" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Account Login" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="On/Off" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Level Control" client="true" server="false" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Wake on LAN" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Channel" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Target Navigator" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Media Input" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Low Power" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Audio Output" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Application Launcher" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Application Basic" client="true" server="false" clientLocked="true" serverLocked="true"></include>
+        </clusters>
+    </deviceType>
+    <deviceType>
+        <name>MA-casting-videoclient</name>
```

Comment:
Why are there two different (?) MA-casting-videoclient devices?

Reply:
I've forgot to update the name. Renamed it now to: https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/VideoRemoteControl.adoc